client/packages/common/src/utils/numbers/NumUtils.ts

```diff
@@ -1,6 +1,11 @@
 const constrain = (value: number, min: number, max: number) =>
   Math.min(Math.max(value, min), max);
 
+const SECOND = 1000;
+const MINUTE = 60 * SECOND;
+const HOUR = 60 * MINUTE;
+const DAY = 24 * HOUR;
```

Comment:
Maybe just add comment above clarifying that this is in terms of "milliseconds":

```ts
// from milliseconds:
const SECOND = 1000;
const MINUTE = 60 * SECOND;
const HOUR = 60 * MINUTE;
const DAY = 24 * HOUR;
```

Reply:
Oh, I see you kind of did that below. No biggie either way.  :)

Reply:
The reason I but the doc comment into NumUtils is that VSCode doesn't show it when hovering NumUtils.DAY when putting it on these "helpers".

![image](https://user-images.githubusercontent.com/88299195/207720237-5d154bac-c0cf-474a-980d-66525f78d139.png)